### PR TITLE
Aha/audits

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/biz/jdbc/tenants/ProfileController.java [hunk 1]
```diff
@@ -67,8 +72,17 @@ public void profileChangePassword(WebContext ctx) {
...
                 userAccount.getLogin().setCleartextPassword(newPassword);
                 oma.update(userAccount);
+
+                auditLog.neutral("%s changed her or his password.", UserContext.getCurrentUser().getUserName())
+                        .forCurrentUser()
+                        .log();
+
                 showSavedMessage();
             } catch (Exception e) {
+                auditLog.neutral("Ann error occurred when %s tried to changed her or his password: %s",
```
Comment (fkrone) [inline src/main/java/sirius/biz/jdbc/tenants/ProfileController.java]: Typo

File: src/main/java/sirius/biz/jdbc/tenants/ProfileController.java [hunk 2]
```diff
@@ -67,8 +72,17 @@ public void profileChangePassword(WebContext ctx) {
                 userAccount.getLogin().verifyPassword(newPassword, confirmation, userAccount.getMinPasswordLength());
                 userAccount.getLogin().setCleartextPassword(newPassword);
                 oma.update(userAccount);
+
+                auditLog.neutral("%s changed her or his password.", UserContext.getCurrentUser().getUserName())
```
Comment (fkrone) [inline src/main/java/sirius/biz/jdbc/tenants/ProfileController.java]: his or her

File: src/main/java/sirius/biz/protocol/AuditLogController.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import sirius.web.security.UserContext;
+
+/**
+ * Provides an admin GUI to display {@link AuditLogEntry audit log entries}.
+ */
+@Register(classes = Controller.class)
+public class AuditLogController extends BizController {
+
+    @Part
+    private Mongo mongo;
```
Comment (fkrone) [inline src/main/java/sirius/biz/protocol/AuditLogController.java]: Not used?

File: src/main/resources/biz_de.properties [hunk 4]
```diff
@@ -189,6 +189,7 @@ VirtualObject.download=Herunterladen
 VirtualObject.physicalKey=Physikalischer Schl\u00FCssel
 VirtualObject.replaceContents=Inhalt ersetzen
 UserAccountConroller.passwordGenerated=Ein Passwort wurde erzeugt und an '${email}' versendet.
+UserAccountConroller.cannotGeneratePasswordForOwnUser=Sie k\u00F6nnen f\u00FCr Ihren eigenen Benutzer kein Passwort generieren. Verwenden Sie die Funktion 'Passwort \u00E4ndern'.
```
Comment (fkrone) [inline src/main/resources/biz_de.properties]: UTF-8!

File: src/main/resources/default/templates/protocol/audit_logs.html.pasta [hunk 5]
```diff
@@ -0,0 +1,60 @@
+<i:arg type="Page" name="messages"/>
+<w:page titleKey="AuditLogEntry.plural">
+    <i:block name="breadcrumbs">
+        <li><a href="/audit-logs">@i18n("AuditLogEntry.plural")</a></li>
+    </i:block>
+
+    <w:pageHeader titleKey="AuditLogEntry.plural"/>
+
+    <div class="row">
```
Comment (tbiScireum) [inline src/main/resources/default/templates/protocol/audit_logs.html.pasta]: div.row + div.col-md-12 are not necessary

File: src/main/java/sirius/biz/jdbc/tenants/ProfileController.java (hunk 1 above)
Comment (Tobsucht) [reply src/main/java/sirius/biz/jdbc/tenants/ProfileController.java]: einfach ".. tried to change the password" ?

File: src/main/java/sirius/biz/jdbc/tenants/TenantController.java [hunk 6]
```diff
@@ -156,7 +161,7 @@ public void tenantUpdate(WebContext ctx, JSONStructuredOutput out, String tenant
             tenant.getPermissions().getConfig();
         }
         tenant.getPermissions().getPermissions().clear();
-        for (String permission : ctx.getParameters("permissions")) {
+        for (String permission: ctx.getParameters("permissions")) {
```
Comment (Tobsucht) [inline src/main/java/sirius/biz/jdbc/tenants/TenantController.java]: format

**Pull request merged**